packages/react-server/src/ReactFizzServer.js

```diff
@@ -2477,6 +2511,17 @@ function trackPostpone(
 
   const keyPath = task.keyPath;
   const boundary = task.blockedBoundary;
+
+  if (boundary === null) {
+    segment.id = request.nextSegmentId++;
+    trackedPostpones.rootSlots = segment.id;
+    if (request.completedRootSegment !== null) {
+      // Postpone the root if this was a deeper segment.
+      request.completedRootSegment.status = POSTPONED;
+    }
+    return;
+  }
+
```

Comment:
There is a reference below to
```
throw new Error(
  'It should not be possible to possible to postpone at the root...
 )
```

This whole boundaryKeyPath existence check can go away or if you're concerned it could still happen in maybe we just make the message clear it isn't about postponing at the root

Reply:
The "root" in that case means the very first React node before entering the first element.

I'm not sure if that's actually impossible. I think it might be if you pass in a `React.lazy` at the root which then postpones in the reject.

Reply:
ah ty